dye/tasklib/django.py

```diff
@@ -205,6 +205,13 @@ def update_db(syncdb=True, drop_test_db=True, force_use_migrations=True, databas
 
         django_version = _get_django_version()
 
+        if django_version[0] >= 1 and django_version[1] >= 8:
+            # django 1.7 always checks whether migrations are done and fakes
+            # them if they are. For 1.8, you need the --fake-initial flag to
+            # achieve this, at least the first time round
+            _manage_py(['migrate', '--noinput', '--fake-initial', '--no-initial-data'])
+            # then with initial data, AFTER tables have been created:
+            _manage_py(['migrate', '--noinput'])
         if django_version[0] >= 1 and django_version[1] >= 7:
```

Comment:
the second and third `if` should be `elif` - we only want one branch to be executed here.